Classes/Controller/CalendarController.php

```diff
@@ -49,6 +49,7 @@ class CalendarController extends \TYPO3\CMS\Extbase\Mvc\Controller\ActionControl
 	 */
 	public function listAction($addids = array(), $eventId = null) {
 		$cssFile = $GLOBALS['TSFE']->tmpl->getFileName($this->settings['cssFile']);
+		$htmlFile = $GLOBALS['TSFE']->tmpl->getFileName($this->settings['htmlFile']);
```

Comment:
Tamplate-Switch über Typoscript konfigurierbar, weil ich das Plugin gerne in verschiedenen Arten einbinden möchte.

Reply:
jo alles super .. tut das denn auch, wenn ich es nicht setzte? also wie ist die regression

Reply:
ja, dann wird in Zeile 21 der Configuration/TypoScript/constants.txt "htmlFile = EXT:sh_scoutnet_kalender/Resources/Private/Templates/Calendar/List.html" nicht überschieben und geladen.

Reply:
ahh top .. genau, was man haben will, hab es nicht genau verfolgt, ob das default gesetzt war ..